app/src/org/commcare/google/services/analytics/CCPerfMonitoring.kt

```diff
@@ -0,0 +1,20 @@
+package org.commcare.google.services.analytics
+
+import com.google.firebase.perf.FirebasePerformance
+import com.google.firebase.perf.metrics.Trace
+
+object CCPerfMonitoring {
+    // Traces
+    // Measures the duration of synchronous case list loading
+    const val SYNC_ENTITY_LIST_LOADING = "sync_case_list_loading"
```

Comment:
opt: would be nice to adapt a naming conventions here that distinguishes between the trace and attributes like - `TRACE_ SYNC_ENTITY_LIST_LOADING` and `ATTR_ NUM_CASES_LOADED`